aries_cloudagent/protocols/out_of_band/v1_0/manager.py

```diff
@@ -126,25 +126,22 @@ async def create_invitation(
             auto_accept
             or (
                 auto_accept is None
-                and self._session.settings.get(
-                    "debug.auto_accept_requests_public"
-                    if public
-                    else "debug.auto_accept_requests_peer"
-                )
+                and self._session.settings.get("debug.auto_accept_requests_explicit")
             )
         )
-        if public and multi_use:
-            raise OutOfBandManagerError(
-                "Cannot create public invitation with multi_use"
-            )
-        if public and accept != self._session.settings.get(
-            "debug.auto_accept_requests_public",
-            False,
-        ):
-            raise OutOfBandManagerError(
-                "Cannot override auto-acceptance configuration for "
-                "requests to invitations on public DIDs"
-            )
+        if public:
+            if multi_use:
+                raise OutOfBandManagerError(
+                    "Cannot create public invitation with multi_use"
+                )
+            if attachments:
+                raise OutOfBandManagerError(
+                    "Cannot create public invitation with attachments"
```

Comment:
According to RFC 0434, an existing connection can only be reused based on a public DID. If we cannot create a public invitation with attachments then it will not support the last 2 cases in the following table:
https://github.com/hyperledger/aries-rfcs/tree/master/features/0434-outofband#responses